languages/erlang/config.json

```
@@ -5,7 +5,15 @@
     "indent_size": 4
   },
   "exercises": {
-    "concept": [],
+    "concept": [
+      {
+        "slug": "basics",
+        "uuid": "e21279ba-ee09-412a-902d-478ccdc5d28c",
+        "prerequisites": [],
+        "topics": ["math", "functions", "variables"],
+        "difficulty": 1
```

We've updated the spec a bit in the meantime. Here's how to update to the latest specs

```suggestion
        "slug": "lucians-luscious-lasagna",
        "name": "Lucian's Luscious Lasagna",
        "uuid": "e21279ba-ee09-412a-902d-478ccdc5d28c",
        "concepts": ["basics"],
        "prerequisites": [],
```

As we're now naming exercises for their story/theme (`lucians-luscious-lasagna` in this case), the directory of the exercise should also be changed to `lucians-luscious-lasagna` (so that would be `languages/erlang/exercises/concept/lucians-luscious-lasagna`).